framework/client/development.rst

```diff
@@ -9,12 +9,23 @@ Development
 Setup
 -----
 
+.. warning::
+  The environment variable ``FONTAWESOME_NPM_AUTH_TOKEN`` has to be available otherwise
+  the setup command will fail.
+  With osx this can be done with:
+
+  .. code-block:: console
+
+    launchctl setenv FONTAWESOME_NPM_AUTH_TOKEN {TOKEN}
+
+  The value of {TOKEN} can be found on sharepoint.
```

Comment:
Perhaps the SharePoint document should be linked. Can be hard to find anything there.